modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -5,46 +5,185 @@
-| `apiVIP` | `api.<clusterdomain>` | The VIP to use for internal API communication.
 
-This setting must either be provided or pre-configured in DNS so that the
-default name resolve correctly.
-| `ingressVIP` | `test.apps.<clusterdomain>` | The VIP to use for ingress traffic.
+| [[apivip]]`apiVIP` | `api.<clustername.clusterdomain>` | The VIP to use for internal API communication.
+
+This setting must either be provided or pre-configured in the DNS so that the
+default name resolves correctly.
+
+| `disableCertificateVerification` | `False` | `redfish` and `redfish-virtualmedia` need this parameter to manage BMC addresses. The value should be `True` when using a self-signed certificate for BMC addresses.
 
-This setting must either be provided or pre-configured in DNS so that the
-default name resolve correctly.
-|`dnsVIP` | | The VIP to use for internal DNS communication.
+| [[ingressvip]]`ingressVIP` | `test.apps.<clustername.clusterdomain>` | The VIP to use for ingress traffic.
+
+ifeval::[{release} < 4.5]
+Provide this setting or pre-configure it in the DNS so that the default name resolves correctly.
+|[[dnsVIP]]`dnsVIP` | | The VIP to use for internal DNS communication.
 
 This setting has no default and must always be provided.
-|===
+endif::[]
 
-.Hosts
+|===
 
-The `hosts` parameter is a list of separate bare metal assets that should be used to build the cluster.
 
+[cols="1,1,3", options="header"]
+.Optional Parameters
 |===
-|Name |Default |Description
-| `name` |  | The name of the `BareMetalHost` resource to associate with the details.
-| `role` |  | Either `master` or `worker`.
-| `bmc` | | Connection details for the baseboard management controller. See below for details.
-| `bootMACAddress` |  | The MAC address of the NIC the host will use to boot on the provisioning network.
+|Parameters
+|Default
+|Description
+
+
+ifeval::[{release} > 4.3]
+|`provisioningDHCPExternal`
+| false
+|Defines if the installer uses an external DHCP or the provisioner node DHCP.
+
+|`provisioningDHCPRange`
+|`172.22.0.10,172.22.0.100`
+|Defines the IP range for nodes on the `provisioning` network.
+
+a|`provisioningNetworkCIDR`
+|`172.22.0.0/24`
+|The CIDR for the network to use for provisioning. This option is required when using IPv6 addressing on the `provisioning` network.
+endif::[]
+
+|`clusterProvisioningIP`
+|The 3rd IP address of the `provisioningNetworkCIDR`
+|The IP within the cluster where the provisioning services run. Defaults to the 3rd IP of the `provisioning` subnet. For example, `172.22.0.3`.
+
+|`bootstrapProvisioningIP`
+|The 2nd IP address of the `provisioningNetworkCIDR`
+|The IP on the bootstrap VM where the provisioning services run while the the installer is deploying the control plane (master) nodes. Defaults to the 2nd IP of the `provisioning` subnet. For example, `172.22.0.2` or `2620:52:0:1307::2`. When using no `provisioning` network, set this value to an IP address that is available on the `baremetal` network.
```

Comment:
drop master here?

Reply:
Ibid, we need to maintain master in certain cases. The config file examples will have it until we get rid of it in the software. So I've made it parenthetical to de-emphasize it.